src/applications/simple-forms/20-10207/pages/nameAndDateofBirth.js

```diff
@@ -0,0 +1,30 @@
+import { fullNameNoSuffixUI } from 'platform/forms-system/src/js/web-component-patterns/fullNamePattern.js';
+
+import {
+  dateOfBirthSchema,
+  dateOfBirthUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+import { pdfFullNameNoSuffixSchema } from '../../shared/definitions/pdfFullNameNoSuffix';
+
+import { getNameAndDobPageTitle } from '../helpers';
+
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
+    // TODO: Use ...titleUI() once that supports functions for title
+    'ui:title': ({ formData }) => getNameAndDobPageTitle(formData),
+    fullName: fullNameNoSuffixUI(),
+    dateOfBirth: dateOfBirthUI({ required: true }),
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      fullName: pdfFullNameNoSuffixSchema({
+        pdfMaxLengths: { first: 12, middle: 1, last: 18 },
```

Comment:
what does the UI do for this? block input or give a character count warning? I wonder if design has recommendation

Reply:
Removed pdf max-lengths and associated code.